README.tfdoc.hcl

```diff
@@ -50,7 +52,11 @@ section {
   section {
     title   = "Module Features"
     content = <<-END
-      A [Terraform] base module for creating `terraform-google-compute-subnetwork` which creates a subnet into a specified VPC. Each VPC network is subdivided into subnets, and each subnet is contained within a single region. You can have more than one subnet in a region for a given VPC network. If no `log_config` is specified `default_log_config` with best practices will be applied.
+      A [Terraform] module for creating `google_compute_subnetwork` resources
+      which create subnets for a specified VPC. Each VPC network is subdivided
+      into subnets, and each subnet is contained within a single region.
+      You can have more than one subnet in a region for a given VPC network.
+      If no `log_config` is specified `default_log_config` with best practices will be applied.
```

Comment:
Should these lines be split up like they are, or can the line breaks be removed where appropriate?

Reply:
I don't think it makes a difference? Markdown will wrap lines anyways...